### PR TITLE
Update free-programming-cheatsheets.md

### PR DESCRIPTION
**Pull request by Lisaluu555**
## What does this PR do?
Add resource(s) 

### Description
GraphQL Cheat Sheet

### Why is this valuable (or not)?
Free

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -159,6 +159,7 @@
 ### GraphQL
 
 * [GraphQL Cheatsheet](https://devhints.io/graphql) - devhints, Rico Santa Cruz (HTML)
+* [GraphQL Cheat Sheet](https://simplecheatsheet.com/tag/graphql-cheat-sheet/) - SIMPLE CHEAT SHEET (HTML)
```
Comment (shrn01) [inline more/free-programming-cheatsheets.md:162]:
```suggestion
* [GraphQL Cheat Sheet](https://simplecheatsheet.com/tag/graphql-cheat-sheet) - Simple Cheat Sheet (HTML)
```

**Review by shrn01 — CHANGES_REQUESTED** [review]

**Comment by shrn01** [comment]
Please collaborate with #8547

File: more/free-programming-cheatsheets.md [hunk 2]
```diff
@@ -159,6 +159,7 @@
 ### GraphQL
 
 * [GraphQL Cheatsheet](https://devhints.io/graphql) - devhints, Rico Santa Cruz (HTML)
+* [GraphQL Cheat Sheet](https://simplecheatsheet.com/tag/graphql-cheat-sheet) - Simple Cheat Sheet (HTML)
```
Comment (shrn01) [inline more/free-programming-cheatsheets.md:162]: Alphabetically order the entries. Here swap line 161 and 162

Comment (Lisaluu555) [reply more/free-programming-cheatsheets.md:161]: Solved

**Pull request merged**